### PR TITLE
Routing setup

### PR DESCRIPTION
**Pull request by accolitelvr**
Setup of routing, including dynamic generation of routing and toolbar from a service. Adding of components.

### CODE REVIEW

File: src/app/components/tfrs/tfrs.component.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-tfrs',
+  templateUrl: './tfrs.component.html',
+  styleUrls: ['./tfrs.component.scss']
+})
+export class TfrsComponent implements OnInit {
+  TfrId!: String | null;
```
Comment (prasadkorrapati) [inline src/app/components/tfrs/tfrs.component.ts:10]: TFRId should be number as below
TfrId !: number;

Move all the logic to TFRComponent, and not here

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
+  RouteList = RoutesService.RouteList;
+  constructor() {}
   title = 'TFR-Management';
```
Comment (prasadkorrapati) [inline src/app/app.component.ts:12]: This line is not required.

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+import { MilestonesComponent } from './components/milestones/milestones.component';
+import { PermissionsComponent } from './components/permissions/permissions.component';
+import { ReportingComponent } from './components/reporting/reporting.component';
+import { TfrsComponent } from './components/tfrs/tfrs.component';
+import { TfrComponent } from './components/tfr/tfr.component';
+import { RoutesService } from './routes.service';
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forRoot(RoutesService.RouteList)],
```
Comment (prasadkorrapati) [inline src/app/app-routing.module.ts:13]: Looks good

**Review by prasadkorrapati — COMMENTED** [review]
Please address review comments

**Comment by accolitelvr** [comment]
Have made requested changes @prasadkorrapati

**Pull request merged**